### PR TITLE
chore: small edits

### PR DESCRIPTION
**Pull request by Coniferish**
Add comments and fix decorators on some tests.

### CODE REVIEW

File: test_unstructured/staging/test_weaviate.py [hunk 1]
```diff
@@ -53,6 +54,7 @@ def test_stage_for_weaviate(filename="example-docs/layout-parser-paper-fast.pdf"
     }
 
 
+@pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
```
Comment (scanny) [inline test_unstructured/staging/test_weaviate.py:57]: I know there are a lot like this, but this reason is not a reason, it just restates the "skip_outside_ci" label. A reason would answer the question "Why do we skip this when we're not running in the CI environment?"

File: test_unstructured/staging/test_weaviate.py [hunk 2]
```diff
@@ -17,9 +17,10 @@
 )
 
 is_in_docker = os.path.exists("/.dockerenv")
+skip_outside_ci = os.getenv("CI", "").lower() in {"", "false", "f", "0"}
```
Comment (scanny) [inline test_unstructured/staging/test_weaviate.py:20]: This is a predicate (resolves to True or False) but is named like a function.

I'd say something like "is_in_ci" would be a better name, flip the logic to "not in" and then use `not is_in_ci` in the `skipif` expression.

File: requirements/deps/constraints.txt [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ####################################################################################################
 # This file can house global constraints that aren't *direct* requirements of the package or any
 # extras. Putting a dependency here will only affect dependency sets that contain them -- in other
-# words, if something does not require a constraint, it will not be installed.
+# words, if something does not require a constraint, it will not be installed. Refer to the "Adding
+# a new Dependency Group" notes in Notion for more info about managing requirements.
```
Comment (scanny) [inline requirements/deps/constraints.txt:5]: Open-source users aren't going to have access to the internal Notion. Not sure exactly what to do about that but something to think about.

**Review by scanny — CHANGES_REQUESTED** [review]
A few suggested changes.

File: requirements/deps/constraints.txt (hunk 3 above)
Comment (Coniferish) [reply requirements/deps/constraints.txt:5]: That did cross my mind and I was wondering if we should just add a README or something with the contents of that Notion page. It's just general info that can be a good reference, even for potential contributors. Lmk if you have any more thoughts or if we should bring someone else to weigh in

Comment (scanny) [reply requirements/deps/constraints.txt:5]: I'd ask Matt and let him make the call.

File: test_unstructured/partition/test_api.py [hunk 4]
```diff
@@ -102,7 +102,7 @@ def test_partition_via_api_raises_with_bad_response(request: FixtureRequest):
     partition_mock_.assert_called_once()
 
 
-@pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
+@pytest.mark.skipif(not is_in_ci, reason="Skipping test run outside of CI")
```
Comment (scanny) [inline test_unstructured/partition/test_api.py:105]: Are the rest of these skipped for the same reason?

Comment (Coniferish) [reply test_unstructured/partition/test_api.py:105]: I don't know. I didn't look into it

File: requirements/deps/constraints.txt (hunk 3 above)
Comment (Coniferish) [reply requirements/deps/constraints.txt:5]: @MthwRobinson ?

Comment (MthwRobinson) [reply requirements/deps/constraints.txt:5]: I'd leave references to our internal Notion out of the open source repo.

**Review by scanny — APPROVED** [review]
k, good enough for me. Maybe ping Matt about those non-reason reasons and we can clarify those later.

**Pull request merged**